content.js

```diff
@@ -578,7 +578,7 @@ function changeStartSponsorButton(showStartSponsor, uploadButtonVisible) {
   if (showStartSponsor) {
     showingStartSponsor = true;
     document.getElementById("startSponsorImage").src = chrome.extension.getURL("icons/PlayerStartIconSponsorBlocker256px.png");
-    document.getElementById("startSponsorButton").setAttribute("title", "Sponsor Starts Now");
+    document.getElementById("startSponsorButton").setAttribute("title", browser.i18n.getMessage("SponsorStart"));
```

Comment:
We are using chrome.* everywhere, so I think we should stick with one to be consistent.

Reply:
What do you mean?

Reply:
Instead of `browser.*`

Reply:
Ah ok will change

Reply:
Changed to chrome